lib/OpenQA/Utils.pm

```diff
@@ -257,6 +258,41 @@ sub run_cmd_with_log {
     return run_cmd_with_log_return_error($cmd)->{status};
 }
 
+sub run_cmd_with_log_return_all {
+    my ($cmd) = @_;
+    log_info("Running cmd: " . join(' ', @$cmd));
+    try {
+        my ($stdout, $stderr);
+        my $ipc_run_succeeded = IPC::Run::run($cmd, \undef, \$stdout, \$stderr);
+        my $return_code       = $?;
+        chomp $stdout;
+        chomp $stderr;
+        if ($ipc_run_succeeded) {
+            log_debug($stdout);
+            log_info("cmd returned $return_code");
```

Comment:
In the context this code runs (the web UI service) this log message is not very meaningful and raises the question what command it is about. Just printing `$stdout` without context is also not very meaningful. I suppose it is sufficient to return that information back to the user.